### ISSUE TITLE
Accessibility review failing - it looks like contrast ratio is the cause (as it is in Error! mode) - both latest Chrome and Canary,  Win10

### ISSUE THREAD

**Issue by BogdanCerovac** [GoogleChrome/lighthouse#11384]
Tested locally, on two different Windows 10, both evergreen Chrome (Version 85.0.4183.83 (Official Build) (64-bit)) and Canary (Version 87.0.4254.0 (Official Build) canary (32-bit)).

Axe did not have any problems v4.5.3 (axe-core 3.5.5)

#### Provide the steps to reproduce
1. Run LH on:
- https://www.nrk.no/sorlandet/fargesvake-ser-ikke-rod-t--onsker-seg-gul-1.15125048
- https://www.nrk.no
  
#### What is the current behavior?
Accessibility KPI gets a question mark, Contrast check seems to be the cause as it falls into Error state
#### What is the expected behavior?
Accessibility KPI should get a point, Contrast check should report some errors (according to Axe)

#### Environment Information
* Affected Channels: DevTools 
* Lighthouse version: Lighthouse 6.0.0 and Lighthouse 6.3.0
* Chrome version:  both evergreen Chrome (Version 85.0.4183.83 (Official Build) (64-bit)) and Canary (Version 87.0.4254.0 (Official Build) canary (32-bit))
* Node.js version: NA
* Operating System: Windows 10 Pro and Home edition

**Related issues**
Potential cause either contrast eval or maybe missing manifest.json, no other messages in the console

**Comment by patrickhulce** [comment]
It's a bug in axe-core see https://github.com/dequelabs/axe-core/issues/2483 for progress.

**Comment by BogdanCerovac** [comment]
@patrickhulce - thank you!

**Comment by sergeykish** [comment]
I've got same problem, very confusing — low contrast ratio with default colors http://sergeykish.com/web-api-element-prototype-queryxpathall

Reproduced original issue. Issue was not found by "Background and foreground colors have a sufficient contrast ratio Error!", maybe now it will.

**Comment by anisse** [comment]
Hi, my website is affected by this issue:

https://anisse.astier.eu

https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fanisse.astier.eu

It also seems #12613 might be a duplicate.

**Comment by connorjclark** [comment]
Should have been resolved long ago. Can't repro today.
